platform/core/src/classes/ImageSet.ts

```diff
@@ -79,63 +75,96 @@ class ImageSet {
     return this.images[index];
   }
 
+  /**
+   * Default image sorting. Sorts by the following (in order of priority)
+   * 1. Image position (if ImagePositionPatient and ImageOrientationPatient are defined)
+   * 2. InstanceNumber
+   * Note: Images are sorted in-place and a reference to the sorted image array is returned.
+   *
+   * @returns images - reference to images after sorting
+   */
+  sort(sortingCallback: (a: Image, b: Image) => number): Image[] {
+    try {
+      return this.sortByImagePositionPatient();
+    } catch (error) {
```

Comment:
Catching an error for a normal flow isn't the right way to handle this - what you should do is something like:
const sortImagePositionCompare = createSortByImagePositionPatient(images);
this.images.sort(sortImagePositionCompare || defaultImageSort)